docs/reference/cli/README.mdx

```diff
@@ -100,7 +100,50 @@ as [client intents file(s)](/intents-and-intents-files/#intents-file-formats) in
 
 #### Returns
 
-Here's a partial output from `otterize mapper export -n otterize-ecom-demo`:
+Here's a partial output from `otterize network-mapper export -n otterize-ecom-demo`:
+
+```shell
+apiVersion: k8s.otterize.com/v1alpha2
+kind: ClientIntents
+metadata:
+  name: cartservice
+  namespace: otterize-ecom-demo
+spec:
+  service:
+    name: cartservice
+  calls:
+    - name: redis-cart
+      type: http
+---
+apiVersion: k8s.otterize.com/v1alpha2
+kind: ClientIntents
+metadata:
+  name: checkoutservice
+  namespace: otterize-ecom-demo
+spec:
+  service:
+    name: checkoutservice
+  calls:
+    - name: cartservice
+      type: http
+    - name: currencyservice
+      type: http
+```
+
+### `otterize network-mapper intents-convert [-f PATH]`
```

Comment:
This might be removed, under discussion. It still exists in the CLI so keeping the documentation for now.